### ISSUE TITLE
changes pages has pagination bug

### ISSUE THREAD

**Issue by nathandunn** [GMOD/Apollo#805]
A user reported that paging through row counts on the pagination page, they were receiving changing numbers of rows:

```
Paging through I get the following row counts 5,2,4,3,3,3,3,3,4,4
```

This is probably because the "count" code does not agree with the "list" code.

**Comment by cmdcolin** [comment]
This might be 2.0.0 instead of 2.0.1? I had noted the issue with inconsistent row counts in 2.0.0 in #642

**Comment by nathandunn** [comment]
Probably.  Most likely was just never fixed.  It is assigned to 2.0.3 now.

**Comment by cmdcolin** [comment]
If they're using 2.0.1 it would be fixed, I pretty much guarantee it :)

**Comment by nathandunn** [comment]
Confirmed .. this was fixed in 2.0.1 . .  @strumpfe  . . .
